extension/src/configuration.ts

```diff
@@ -10,7 +10,7 @@ export class AvmDebugConfigProvider
   implements vscode.DebugConfigurationProvider
 {
   /**
-   * Massage a debug configuration just before a debug session is being launched,
+   * Message a debug configuration just before a debug session is being launched,
```

Comment:
```suggestion
   * Massage a debug configuration just before a debug session is being launched,
```

I think massage is the correct term here.